lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -1025,12 +1025,6 @@ struct LowerVectorAddOrSubOpToAIEVecAddElemOrSubElemOp
           unsigned rBitWidth =
               rSrcType.getElementType().getIntOrFloatBitWidth();
 
```

Comment:
Line 1023-1026 can be deleted.

Reply:
done and get rid of the compilation warning.